docs/DeveloperGuide.md

```diff
@@ -860,7 +860,7 @@ testers are expected to do more *exploratory* testing.
 
 2. The current `linkcontact` command does not recognise an inputted phone number if there are extra spaces around it. For example, `linkcontact 1      12345678` is not allowed. We plan to allow the extra spaces by ignoring extra spaces when splitting the input arguments.
 
-3. upon using the current `linkcontact` command on a contact that is already linked to a specified event, the application returns "contact successfully linked" message in the message display box. We plan to change the display message to "Contact already linked" instead for contacts that are already linked.
+3. Upon using the current `linkcontact` command on a contact that is already linked to a specified event, the application returns a 'Successfully linked' message in the message display box. We plan to change the display message to 'Contact already linked' instead for contacts that are already linked.
```

Comment:
I think that the messages should be in`` instead of '' to be consistent with the message in point 1.

Reply:
fixed!